app/javascript/packs/Components/CuisineMap.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+ const cuisineMap = {
+    'asian':'a',
+    'tes':'t'
+};
+
+export default cuisineMap;
```

Comment:
You can write this as:
```js
export default {
  asian: 'a',
  tes: 't',
};
```

There's no need to `import React` if you don't define a component or use React directly.
There's also no need to wrap the object keys with quotes if they don't contain special characters (like dashes, for example).